### ISSUE TITLE
Update ether.js helper (web3.toWei is not a function, BigNumber is not a constructor)

### ISSUE THREAD

**Issue by JohannesMayerConda** [OpenZeppelin/openzeppelin-solidity#1593]
I'm currently upgrading to solidity v0.5 and have it compiling already but my tests don't run through because of a test helper I use from openzeppelin. Concretely the file /test/helpers/ether.js

**💻 Environment**
openzeppelin-solidity (package.json): ^2.1.1
web3 (package.json): ^1.0.0-beta.37
truffle (package.json and global): v5.0.1 with solc-js v0.5.0

**🔢 Code to reproduce bug**
In a test file (stripped down version that should do essentially the same, didn't try it tough)
import { ether } from 'openzeppelin-solidity/test/helpers/ether'
contract('TEST', (accounts) => {
        it('will fail', async () => {
                const cap = ether("30") //fails
        })
}

I can fix it changing the ether.js file to:
function ether (n) {
  return new web3.utils.BN(web3.utils.toWei(n, 'ether'));
}

module.exports = {
  ether,
};

I could contribute this fix but I guess there might be a reason this is still in the old syntax and the team is aware of the "problem"... Or am I doing something wrong?

PS: constants.js same problem

PPS: Also noticed in time.js all web3.currentProvider.sendAsync should be just web3.currentProvider.send

**Comment by nventuro** [comment]
Hey there @JohannesMayerConda! During our own migration to Solidity 0.5.0, I actually started on these helpers: there are indeed many web3 1.0 and truffle 5 changes that break them.

Now that they're fixed, I migrated them to a separate repository and created an npm package with them, so that they can be used by everyone without having to extract the code from OpenZeppelin's test suite: https://github.com/OpenZeppelin/openzeppelin-test-helpers.

It is still an early effort, and some improvement is due on the API, error messages, etc., but it does work, and will hopefully be useful to you!

**Comment by JohannesMayerConda** [comment]
Haha, oh nose I needed this a few days ago... Meanwhile I copied and adapted the helpers myself. 

But I will switch to the new helpers in the future! Anyway - thanks, good work! This helps a lot 👍

**Comment by k06a** [comment]
`openzeppelin-solidity 2.1.1` still contains old `constants.js`:
```
const BigNumber = web3.BigNumber;

module.exports = {
  ZERO_ADDRESS: '0x0000000000000000000000000000000000000000',
  MAX_UINT256: new BigNumber(2).pow(256).minus(1),
  MAX_INT256: new BigNumber(2).pow(255).minus(1),
  MIN_INT256: new BigNumber(2).pow(255).times(-1),
};
```

Tests in my project contains following lines:
```
const { shouldBehaveLikePublicRole } = require('openzeppelin-solidity/test/access/roles/PublicRole.behavior');

...

shouldBehaveLikePublicRole(manager, otherManager, otherAccounts, 'manager');
```

And internally it requires `constants.js`, so I need `2.1.2` with this fix ASAP :)

**Comment by k06a** [comment]
@nventuro are you planning to ship `2.1.2` from current `master`?

**Comment by nventuro** [comment]
Normally I'd say that you shouldn't use our tests in your project, since we have no stability guarantees on those (in fact, I don't think we should be publishing them), though we _do_ have plans to make it so. However, `PublicRole.behavior` is a rather special case, a 2.1.2 may be due for that.

**Comment by nventuro** [comment]
@k06a I've just released a v2.1.2 package, in which only that behavior is published, using the new test helpers. Keep in mind this is an initiative we're only now starting out, so there may be some changes in future releases, but we'll try to keep them to a minimum.

**Comment by k06a** [comment]
@nventuro special thanks!
